src/main/scala/slamdata/pkg/PackageAliases.scala

```diff
@@ -14,10 +14,12 @@
  * limitations under the License.
  */
 
-package quasar.pkg
+package slamdata.pkg
 
-import scala.collection.{ mutable => scm, immutable => sci }
 import java.lang.SuppressWarnings
+import scala.{collection => sc}
+import scala.collection.{ mutable => scm, immutable => sci }
+import scala.{util => su}
 
 @SuppressWarnings(scala.Array("org.wartremover.warts.MutableDataStructures"))
```

Comment:
If you check out the edits for this file (and PackageMethods) in https://github.com/quasar-analytics/quasar/compare/master...sellout:sql-distinct#diff-91eff94d671cb9207858b1d3f4d061d0 you can get rid of this Suppression and a bunch of other things that shouldn’t be in our Predef.